packages/permission-controller/src/PermissionController.ts

```diff
@@ -1477,88 +1472,49 @@ export class PermissionController<
       permission.caveats.splice(caveatIndex, 1);
     }
 
-    this.validateModifiedPermission(permission, origin, target);
+    this.validateModifiedPermission(permission, origin);
```

Comment:
Seems like we could remove `target` from the arguments of `deleteCaveat`? Not sure why it needs to be specified when we have the permission object.

Reply:
Unless there is a good reason I am missing.

Reply:
Done in: [09c3171](https://github.com/MetaMask/core/pull/1337/commits/09c3171cbe0ca95266a6a5b8953e96e2c60bca3a)